api/Document.json

```diff
@@ -867,10 +867,10 @@
               "version_added": "5"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "≤37"
```

Comment:
This is the only irregular change in this PR. It probably was in earlier WebView, but let's not chase down that version number.